domain/robotouille.json

```diff
@@ -170,6 +170,10 @@
         {
             "name": "container_at",
             "param_types": ["container", "station"]
+        },
+        {
+            "name": "ispan",
+            "param_types": ["item"]
```

Comment:
Remove this predicate, and remember to include the "iswatermelon" and "issteak" predicates, since we will need those predicates for when we make recipes and check if customers' orders have been satisfied